crnk-core/src/test/java/io/crnk/core/engine/document/DocumentTest.java

```diff
@@ -18,7 +21,11 @@ public class DocumentTest {
 
 	@Test
 	public void testDocumentEqualsContract() {
-		EqualsVerifier.forClass(Document.class).usingGetClass().suppress(Warning.NONFINAL_FIELDS).verify();
+		EqualsVerifier.forClass(Document.class).usingGetClass()
+				// https://github.com/jqno/equalsverifier/issues/486
+				.withPrefabValues(JsonNode.class, NullNode.instance, new TextNode("foo"))
+				.withIgnoredFields("jsonapi") // ignore unused fields in equals and hashcode
+				.suppress(Warning.NONFINAL_FIELDS).verify();
```

Comment:
We need to find the ones we are changing and format them the same way - like it was done in https://github.com/tory-kk/crnk-framework/pull/1/files#diff-f787aeb0a6d3c86e24302d0f55c88e867c4544d0b110ffd7d2d8796c46093599

To be fair, Sergey, tried doing it in his PR but it was creating a diff relative to your PR which was purely reformatting thing so I asked him not to do that.

But looking at your PR now, lets stick to operation per line for these cases